examples/bin/run-java

```diff
@@ -40,6 +40,11 @@ then
     --add-opens=java.base/java.lang=ALL-UNNAMED \
     --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED \
     "$@"
+elif [ "$JAVA_MAJOR" != "" ] && [ "$JAVA_MAJOR" -ge "11" ]
+then
+  exec "$JAVA_BIN" \
+    --add-exports=java.base/jdk.internal.ref=ALL-UNNAMED \
```

Comment:
Let's add all 4 options described in the "if you are running Java 11-13" section of datasketches-memory docs: https://github.com/apache/datasketches-memory#use-as-a-library-using-jars-from-maven-central

```
--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED
--add-exports=java.base/jdk.internal.ref=ALL-UNNAMED
--add-opens=java.base/java.nio=ALL-UNNAMED
--add-opens=java.base/sun.nio.ch= ALL-UNNAMED
```

And include a comment here that these parameters are for datasketches-memory.

Reply:
Added